aws/table_aws_ecs_cluster.go

```diff
@@ -0,0 +1,283 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ecs"
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsEcsCluster(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_ecs_cluster",
+		Description: "AWS ECS Cluster",
+		Get: &plugin.GetConfig{
+			KeyColumns:        plugin.SingleColumn("cluster_arn"),
+			ShouldIgnoreError: isNotFoundError([]string{"ResourceNotFoundException", "InvalidParameterException"}),
+			Hydrate:           getEcsCluster,
+		},
+		List: &plugin.ListConfig{
+			Hydrate: listEcsClusters,
+		},
+		GetMatrixItem: BuildRegionList,
+		Columns: awsRegionalColumns([]*plugin.Column{
+			{
+				Name:        "cluster_arn",
+				Description: "The Amazon Resource Name (ARN) that identifies the cluster.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("ClusterArn"),
+			},
+			{
+				Name:        "cluster_name",
+				Description: "A user-generated string that you use to identify your cluster.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getEcsCluster,
+				Transform:   transform.FromField("ClusterName"),
+			},
+			{
```

Comment:
Don't need to add `transform.FromField()` if the camelCase value of column name is same as SDK property.